eng/scripts/Prepare-APIReview.ps1

```diff
@@ -0,0 +1,48 @@
+Param(
+    [Parameter(Mandatory=$True)]
+    [array] $ArtifactList,
+    [Parameter(Mandatory=$True)]
+    [string] $OutPath,
+    [Parameter(Mandatory=$True)]
+    [string] $ApiKey,
+    [Parameter(Mandatory=$True)]
+    [string] $SourceBranch,
+    [Parameter(Mandatory=$True)]
+    [string] $DefaultBranch,
```

Comment:
You don't need these args to generate token file. It's only required to trigger `Create-APIReview.ps1`